From 169bba6231ee2a816898b0a6d3c082d555f8d1be Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Wed, 28 Jun 2017 16:10:03 -0700
Subject: [PATCH] IMGAPI-632 URL encoded query string induces illegal
 instruction in jsSprintf Reviewed by: Todd Whiteman
 <todd.whiteman@joyent.com> Approved by: Todd Whiteman
 <todd.whiteman@joyent.com>

---
 CHANGES.md   | 4 ++++
 package.json | 4 ++--
 2 files changed, 6 insertions(+), 2 deletions(-)

diff --git a/CHANGES.md b/CHANGES.md
index a0260b4..a6d9e38 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,5 +1,9 @@
 # IMGAPI changelog
 
+## 3.3.1
+
+- IMGAPI-632 Fix potential crash from in static served URLs.
+
 ## 3.3.0
 
 - IMGAPI-627 With this change, a standalone IMGAPI will use an ECDSA key for
diff --git a/package.json b/package.json
index f5bd79f..c6e7847 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "imgapi",
   "description": "Image API to manage images for SDC 7",
-  "version": "3.3.0",
+  "version": "3.3.1",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
@@ -32,7 +32,7 @@
     "passwd": "0.0.11",
     "posix-getopt": "1.0.0",
     "progbar": "git+https://github.com/trentm/node-progbar.git#a4c56e6",
-    "restify": "4.3.0",
+    "restify": "4.3.1",
     "rimraf": "2.2.6",
     "sdc-clients": "10.0.4",
     "semver": "3.0.1",
-- 
2.21.0

